conf/layer.conf

```diff
@@ -35,5 +35,8 @@ CLANGSDK ??= "0"
 
 LLVMVERSION = "18.1.8"
 
+# introduced in oe-core 8905639d breaks the handling of runtime exception when build with clang
+TUNE_CCARGS:remove:toolchain-clang = "${@bb.utils.contains('TUNE_FEATURES', 'aarch64', '-mbranch-protection=standard', '', d)}"
+
```

Comment:
lets move this to `nonclangable.conf`

Reply:
and check if https://github.com/kraj/meta-clang/pull/999 made any difference

Reply:
I have tested it and unfortunately don't solve the issue, also reported that on https://github.com/kraj/meta-clang/issues/998